pkg/meta/tkv_fdb.go

```diff
@@ -0,0 +1,192 @@
+//go:build linux
+// +build linux
+
+/*
+ * JuiceFS, Copyright 2021 Juicedata, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package meta
+
+import (
+	"strings"
+
+	fdb "github.com/apple/foundationdb/bindings/go/src/fdb"
+)
+
+func init() {
+	Register("fdb", newKVMeta)
+	drivers["fdb"] = newFdbClient
+}
+
+const DEBUG = true
+
+type fdbTxn struct {
+	t fdb.Transaction
+}
+
+type fdbClient struct {
+	client fdb.Database
+}
+
+func newFdbClient(address string) (tkvClient, error) {
+	args := strings.Split(address, ":")
+	err := fdb.APIVersion(610)
+	if err != nil {
+		logger.Errorf("Unable to set API version: %v\n", err)
```

Comment:
No need to log here, instead returns the error, like:
`return nil, fmt.Errorf("set API version: %s", err)`